test/gst-msdk/util.py

```diff
@@ -31,6 +31,7 @@ def get_supported_format_map():
     "444P"  : ("y444", "Y444"),
     "P010"  : ("p010-10le", "P010_10LE"),
     "P210"  : ("i422-10le", "I422_10LE"),
+    "Y210"  : ("Y210", "Y210"),
```

Comment:
We just went over this in the other patch.  The first value should be lowercase!

Reply:
thanks, let me also to update on this side by following the other patch's review